### DISCUSSION TITLE
TypeError: Cannot read property 'name' of null

### DISCUSSION THREAD

**Discussion by ImmanuelSalva** [vercel/next.js#33036] [Help]
so I don't know what happened but I started a build in my vercel-next.js project and for some reason it says

![failureproof](https://user-images.githubusercontent.com/83900884/148244226-84a64133-09b1-4dd3-b317-2bf62ae49a4a.png)

Despite me not fiddling with any of the "[slug].js" Pages nor the preview-post.js page (well I changed it a bit but it's nothing really significant).

This is my PostPreview code:

![failureproof2](https://user-images.githubusercontent.com/83900884/148244595-48d5095d-e020-4145-b052-f0af9b4eb52b.png)

is any of the code wrong or is it next.js that is acting up?

**Comment by wwwdepot** [comment]
What you can do is double check all props passing down to the component are defined. If some props are undefined by design, you need to handle those case inside your component.
